TKGame/TKGame.cs

```diff
@@ -110,6 +110,10 @@ protected override void LoadContent()
             EntityManager.Add(Enemy.Instance);
             EntityManager.Add(Item.Instance);
 
+            EntityManager.Add(Enemy.Instance);
+
+            EntityManager.Add(Enemy.Instance);
+
```

Comment:
Not sure if this is what you intended but calling to add an enemy instance multiple times will just make the enemy update() be executed more frequently.

Reply:
I was having some weird stashed change issues, will be merged out